testutil/compose/auto.go

```diff
@@ -149,8 +154,7 @@ func Auto(ctx context.Context, conf AutoConfig) error {
 		}
 	}
 	if !alertSuccess {
-		log.Error(ctx, "Alerts couldn't be polled", nil)
-		return nil // TODO(corver): Fix this and error
+		return errors.New("alerts couldn't be polled, containers offline")
```

Comment:
containers offline isn't correct. Sure it might be the most common reason, but it isn't the only reason.